### ISSUE TITLE
[Bug]: VNAV couldn't create a path using v speeds higher than -1500

### ISSUE THREAD

**Issue by yukikaze04** [amyinorbit/x-uns-bugs#31]
### Contact Details

sky04@live.com

### Version

1.0.11

### What happened?

# What happened
After reaching cruise, I went to the vnav page, like I always do, to get a vpath and a TOD. I always use -1800 target v/s. When I put that target in, the numbers on the vnav page blinked, like a path was actually calculated, but then almost immediately disappeared, leaving the target v/s field empty again, and no TOD displayed. I tried a few times, but only when I gave it a target v/s of -1500, it worked as expected.

# Expected behaviour
Usually, I'd just get a TOD and a path ready to use, I've never seen the vnav just not calculate a path before like that. Honestly not sure if this is even a bug, or intended behavior.

# Attach your log file here
note: there are two flights in this log, the issue appears in the second one, at the very end of the file
[uns1_log.txt](https://github.com/amyinorbit/x-uns-bugs/files/7708092/uns1_log.txt)
screenshot below taken after vnav accepted the v/s of -1500
![image](https://user-images.githubusercontent.com/75759495/145919396-9ff9eafd-61bf-4b68-8476-cf15994e3b20.png)

**Comment by amyinorbit** [comment]
@yukikaze04 Thanks for the report! Still baffled by this one I have to admit. Can you remember if the FMS gave you any error message when you tried to enter something higher than 1500fpm?

**Comment by yukikaze04** [comment]
Nope, no messages, just a quick flash of the numbers. The only thing that I remember changing is the altitude numbers next to the two waypoints, originally the first one was about x29000, and after putting in the v/s, the numbers changed around to different numbers each time, but as I said earlier, there was no TOD visible either in the FMS or in the plane's ND, and the TGT V/S field reverted back to empty within half a second or so.
